### PR TITLE
Remote: The drawbacks

### PR DESCRIPTION
**Pull request by DuncanFenning**
### Blog title

Remote: The drawbacks and how to overcome them

### Blog description

This blog focuses on the drawbacks of remote working. The things I have come to accept and the things that can be done to limit the barrier to entry for new remote workers.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2016-12-06-remote-drawbacks.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### Collaboration
+
+Collaborating with others when you are remote is always more difficult than if you are physically with the other person. Quick chats, whiteboard explanations and group meetings are an everyday activity in the HX tech team. Ensuring these work as well remotely requires organisation.
+
+Getting the basics right makes remote collaboration a lot easier and more bearable for remote workers. Using the appropriate tools and ensuring the meeting is hosted in an appropriate environment solves most problems. Trying to listen in on a meeting that is being drowned out by background noise is a remote worker's nightmare and ensuring this doesn't happen is fairly straightforward.
+
+However, you can't always plan for the impromptu meetings, in which finding a adequately equipped meeting room is not the top priority. For those physically in the meeting, this isn't a problem. For the remote workers, it can be tiring and lonely, not being able to communicate as effectively.
```
Comment (elliottcrush) [inline _posts/2016-12-06-remote-drawbacks.md]: `an adequately room`?

File: _posts/2016-12-06-remote-drawbacks.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+For group meetings, a high quality microphone is a necessity. Being able to hear everyone in the meeting clearly is a must when there is only one remote member. Often using a low quality mic, you can pick up the people nearest the microphone, but those further away get lost in the background noise filter. This makes it very hard for remote workers to concentrate and understand everything that is being said.
+
+For completely remote meetings, I always recommend purpose built headset. These are cheap and readily available in most office environments and the difference in quality between a purpose built headset and a pair of phone earphones is unbelievable. They drown out background noise and pick up all the conversation. Again, using a non purpose built set of headphones or earphones picks up a lot of background noise.
```
Comment (elliottcrush) [inline _posts/2016-12-06-remote-drawbacks.md]: `headsets`

File: _posts/2016-12-06-remote-drawbacks.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### Missing out on social activities
+
+Something you will have to accept when you are a remote worker is that you won't be taking part in all the social activities that take place in a shared office. These can be easily taken for granted when there is something taking place every week in a bustling office. However, when working from home, you miss these social events.
+
+Since I have moved to working from home I've missed multiple events including BBQs, leaving drinks and meals for ex colleagues, halloween activities and more recently the annual fireworks night. To combat this, I have tried to find social activities outside of my working environment. I've joined a football and basketball team and this keeps me busy 3 nights a week. Even so, there is no real substitute to seeing off a close colleague in person, but those sacrifices are usually few and far between. Unfortunately, this is a trade off that comes with the freedom remote working offers, and that is something I have come to accept.
```
Comment (elliottcrush) [inline _posts/2016-12-06-remote-drawbacks.md]: I think we've hold had one BBQ since you left. No need to hyperbolise mate

**Comment by nikolafilchev** [comment]
I'm happy with this 👍

**Pull request merged**